app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -95,6 +101,13 @@ def callback_for(provider)
       flash[:alert] = user_errors
       redirect_to new_user_registration_url
     end
+  rescue Authentication::Authenticator::PreviouslyBanned => _e
+    # Do we want a DD counter for this?
```

Comment:
Yes I think it would be useful information to track

Reply:
✅

Reply:
@maestromac When I revisited error handling I realized that this will already end up in Datadog under the "identity.errors" key, tag with the error class and including the message. I added an assertion for that in the system spec.